packages/mobile/src/components/user-list/UserCard.tsx

```diff
@@ -56,7 +56,7 @@ export const UserCard = (props: UserCardProps) => {
   return (
     <Paper border='default' onPress={handlePress} {...other}>
       <Avatar source={source} onError={handleError} aria-hidden p='m' pb='s' />
-      <Flex ph='l' pb='s' gap='xs'>
+      <Flex ph='l' pb='s' gap='xs' pointerEvents='none'>
```

Comment:
are there pointer events on mobile?

Reply:
Yes! https://reactnative.dev/docs/view#pointerevents

Essentially I wanted the user name not to be clickable, only the entire card